### PR TITLE
[footer]:rename newsletter link in footer

### PR DESCRIPTION
**Pull request by virgoaugustine**
Rename the newsletter link to `Tilmeld nyhedsbrev`

### CODE REVIEW

File: views/base.html [hunk 1]
```diff
@@ -18,340 +19,340 @@
...
+            {% block content %}{{content}}{% endblock %}
+        </div>
+
+        <!-- Footer -->
+        <footer class="site-footer bg-primary text-white-75 mt-10 py-gutter text-sm">
+            <div class="container mx-auto md:flex md:justify-between">
+                <div class="px-gutter pb-6 md:p-gutter md:w-1/3 md:flex flex-col justify-between">
+                    <a class="block w-24 mb-10 h-10 text-white-75 hover:text-white" href="https://ckan.org/">
+                        <span>Powered by</span> {% include "./partials/ckan-logo.svg" %}
+                    </a>
```
Comment (MuhammadIsmailShahzad) [inline views/base.html:143]: @virgoaugustine just making sure, these are all due to changes in indentation right?

Comment (virgoaugustine) [reply views/base.html:143]: Yes that’s right  @MuhammadIsmailShahzad

Comment (MuhammadIsmailShahzad) [reply views/base.html:143]: ok thank.

LGTM!

**Review by MuhammadIsmailShahzad — APPROVED** [review]
LGTM!

**Pull request merged**